marklogic-client-api/src/test/java/com/marklogic/client/test/datamovement/QueryBatcherTest.java

```diff
@@ -875,7 +875,10 @@ public void maxUrisTestWithQueryTask() {
       
       AtomicInteger counter = new AtomicInteger(0);
       QueryBatcher  queryBatcher = dmManager.newQueryBatcher(new StructuredQueryBuilder().collection("maxUrisTest"));
-      queryBatcher.setMaxBatches(2);
+      
+      int forest_count = queryBatcher.getForestConfig().listForests().length;
+      int maxBatches = forest_count-1;
```

Comment:
Would it be sufficient to set maxBatches to 1?

Reply:
Yes. 
Changing max batches to 1.